Assets/Scripts/PlayerControls.cs

```diff
@@ -148,11 +178,55 @@ void GroundMovement()
 		}
 	}
 
-	Vector3 GetInput()
+	/**
+	 * Raycast forward from the camera's position to detect
+	 * items to interact with.
+	 * 
+	 * This performs RaycastAll, and returns the closest collision.
+	 */
+	bool RaycastForward( out RaycastHit closestHit )
+	{
+		Vector3 camPos = Camera.main.transform.position;
+		Vector3 camForward = Camera.main.transform.forward;
+
+		Debug.DrawRay( camPos, camForward * _interactCheckDistance, Color.yellow, 1.0f, false );
+
+		RaycastHit[] hits = Physics.SphereCastAll(
+			new Ray( camPos, camForward ),
+			_interactCheckRadius,
+			_interactCheckDistance,
+			_cutting.cuttableLayer | _actor.actorResources.buddyLayer );
+
```

Comment:
This is correct. I want the arguments to be on a new line (and indented one more level), because if I kept them in line with method call the line would get too long.